restapi/src/main/java/io/stargate/web/resources/RowResource.java

```diff
@@ -409,7 +409,9 @@ private static Object filterToValue(Object val, String column, Table tableData)
     List<String> values = idFromPath(path);
 
     final List<Column> keys = tableMetadata.primaryKeyColumns();
-    if (keys.size() < values.size()) {
+    boolean notAllPartitionKeys = values.size() < tableMetadata.partitionKeyColumns().size();
```

Comment:
This logic probably needs to be added here as well
https://github.com/stargate/stargate/blob/c4f19aefb2dd9caab0a28107313db02b92ab7f0d/restapi/src/main/java/io/stargate/web/resources/v2/RowsResource.java#L365

Reply:
Done